### PR TITLE
Leaderboard setup

### PR DESCRIPTION
**Pull request by GOlivierNation**
What is implemented in this PR:
*************************************
1. Use Gitflow.
2. Create an npm project with webpack. (If you need a refresher, check the previous project or webpack exercise).
3. Write plain HTML markup with minimum styling (just to make the layout).
4. Use ES6 modules, with import and export.
***********************************************************************************************************************************

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+module.exports = {
+  mode: 'development',
+  devtool: 'inline-source-map',
+  devServer: {
+    static: './dist',
+  },
+  entry: './src/assets/js/index.js',
+  plugins: [
+    new HtmlWebpackPlugin({
+      favicon: './src/assets/images/favicon.ico',
```
Comment (brytebee) [inline webpack.config.js]: - [ ] **Required:** This asset keeps breaking your app, kindly fix it.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @GOlivierNation👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- No linter errors.✔️
- Correctly used Git flow.✔️
- Clean code.✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: webpack.config.js (hunk 1 above)
Comment (GOlivierNation) [reply webpack.config.js]: @brytebee You can check now it is resolved

**Review by Deepakdanger — APPROVED** [review]
Hi @GOlivierNation, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**